test/test_ima_verification.py

```diff
@@ -3,8 +3,8 @@
 Copyright 2020 IBM Corporation
```

Comment:
When we have a big disruptive change like this where we have to continue to support some deprecated features, it's best to leave existing tests as-is as much as possible so we know we haven't broken something important before it's removed. Deprecated still means supported. So maybe the best way is to create a new `test_runtime_verification` script that mirrors this one but uses all of the new options/formats. This will allow use to test both old and new easily and when we fully remove the old path it's as easy as removing the test file. You can even put a comment saying that in the old file.

Reply:
As discussed in the team meeting, we aren't maintaining any legacy tenant <-> verifier APIs, so we shouldn't need to maintain backwards-compatible tests either.